integrations/attachement_test.go

```diff
@@ -13,8 +13,9 @@ import (
 	"net/http"
 	"testing"
 
-	"code.gitea.io/gitea/modules/test"
```

Comment:
Can you keep this import here, but add new line between it and assert import (imports structured as 3 blocks 1. core golang 2. gitea 3. all others)

Reply:
@techknowlogick could it be:

1. stdlib
2. third party
3. gitea

Reply:
Sadly all of the imports follow the scheme, and changing it for some and not all would be inconsistent. Changing all would be difficult because there are open PRs that would conflict with new style.